jsHelper/sidebarConfig.js

```diff
@@ -191,10 +190,14 @@
 		}
 	}
 
-	function removeInteraction() {
+	async function removeInteraction() {
```

Comment:
Revert this also since we're using static properties

Reply:
yeah just now did that totally forgot that one